MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -8541,4 +8542,22 @@ public boolean checkYearlyRetirements() {
         }
         return false;
     }
+
+	/**
+	 * Returns the type of rating method as selected in the Campaign Options dialog. Lazy-loaded for performance.
+	 */
```

Comment:
JavaDoc should probably mention that the default return value is a CampaignOpsReputation object.